rackspace/metrics/v2/ingestion/requests.go

```diff
@@ -0,0 +1,205 @@
+package ingestion
+
+import (
+	"github.com/rackspace/gophercloud"
+	"fmt"
+)
+
+// SendMetrics will ingest the metrics for the tenant associated with RackspaceMetrics.
+func SendMetrics(c *gophercloud.ServiceClient, metrics []MetricData) {
+	var res PostResult
+	reqBody := make([]map[string]interface{}, len(metrics))
+	for i := range metrics {
+		Metric := metrics[i]
+		reqBody[i] = map[string]interface{}{
+			"collectionTime": Metric.CollectionTime,
+			"ttlInSeconds":Metric.TtlInSeconds,
+			"metricValue": Metric.MetricValue,
+			"metricName": Metric.MetricName,
```

Comment:
@rgbkrk Thanks! I'll go ahead and commit this new file. I am not sure about less map[string]interface{}. I am only using these to create request bodies out of defined structs, the way it is done in the Create function as mentioned by @jrperritt in his above comment. Are you saying there is a better way to do it using reflection and extracting tags from defined structs. I am confused now :stuck_out_tongue_winking_eye: I would really appreciate if you give me a gist after modifying a function a little bit so that I can get a better understanding.